src/containers/Header/Header.js

```diff
@@ -24,13 +24,16 @@ class Header extends React.Component {
   render() {
     return (
       <div className="header">
+        <i
+          onClick={this.sidebarToggleHandler}
+          className="fa fa-bars header-hamburger"
+        />
         <Sidebar
           opened={this.state.showSidebar}
```

Comment:
Bad. On top of the render function do:
`const { showSidebar } = this.state.`
and then use it.